modules/swagger-codegen/src/main/resources/Java/pojo.mustache

```diff
@@ -105,4 +105,46 @@ public class {{classname}} {{#parent}}extends {{{parent}}}{{/parent}} {{#seriali
     }
     return o.toString().replace("\n", "\n    ");
   }
+  
```

Comment:
It would be nice to add docstrings to the new methods (e.g. writeToParacel, describeContents).